docs/integrations/sources/cimis.md

```diff
@@ -0,0 +1,36 @@
+# CIMIS
+The California Irrigation Management Information System (CIMIS) is a program unit in the Water Use and Efficiency Branch, Division of Regional Assistance, California Department of Water Resources (DWR) that manages a network of over 145 automated weather stations in California. CIMIS was developed in 1982 by DWR and the University of California, Davis (UC Davis). It was designed to assist irrigators in managing their water resources more efficiently. Efficient use of water resources benefits Californians by saving water, energy, and money.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+| `targets_type` | `string` | Targets Type.  |  |
+| `targets` | `array` | Targets.  |  |
+| `daily_data_items` | `array` | Daily Data Items.  |  |
+| `hourly_data_items` | `array` | Hourly Data Items.  |  |
+| `start_date` | `string` | Start date.  |  |
+| `end_date` | `string` | End date.  |  |
+| `unit_of_measure` | `string` | Unit of Measure.  |  |
+
+To get started, register and request your appKey from the [CIMIS website](https://wwwcimis.water.ca.gov/). You will receive an application key (aka appKey).
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| data | Julian | No pagination | ✅ |  ✅  |
```

Comment:
what the hell is Julian? :D

Reply:
It's the id field apparently. It some form of representation for the day/date. Since each record is for each day, i thought it would be better than using a date field